sos/plugins/ovn_central.py

```diff
@@ -65,12 +86,37 @@ def setup(self):
         ]
 
         schema_dir = '/usr/share/openvswitch'
+        if containerized:
+            if self._container_runtime == 'podman':
+                dst = self.get_command_output(
+                    "podman mount %s" % self._container_name)['output']
+                cmd = "cp " + dst.rstrip()
+            else:
+                cmd = 'docker cp %s:' % self._container_name
+
+            for db in ['nb', 'sb']:
+                cmdline = ("%s/usr/share/openvswitch/ovn-%s.ovsschema "
+                           "/tmp/" % (cmd, db))
+                if self.get_command_output(cmdline)['status'] != 0:
+                    self._log_error("Could not retrieve DB schema file from "
+                                    "container %s", self._container_name)
+            schema_dir = '/tmp'
+            if self._container_runtime == 'podman':
+                self.get_command_output(
+                    "podman umount %s" % self._container_name)
```

Comment:
Absolute nack from me. In addition to the fact that we shouldn't be mounting and unmounting resources in a plugin, there is also no reason to be capturing the output of that mount/unmount operation.

Reply:
> Absolute nack from me. In addition to the fact that we shouldn't be mounting and unmounting resources in a plugin, there is also no reason to be capturing the output of that mount/unmount operation.

There is absolutely a reason to capture the output of the *mount* operation:

# podman mount ovn-dbs-bundle-podman-0
/var/lib/containers/storage/overlay/d1679d89311b4a4fc4909b3126ab6abf1458fd4c1968061105b788867c6da69b/merged

It gives you the location where the filesystem is (actually this is not even a mount as without executing a `podman mount` one can access the filesystem directly in the host). But, again, reading the output would be needed.

For unmouting, how would you suggest doing it if it's not through `get_command_output`?

Reply:
Ah, fair point - I was reading that as `get_cmd_output` and not `get_command_output`, my fault there.